src/parsing/binast-deserializer.cc

```diff
@@ -62,11 +63,12 @@ AstNode* BinAstDeserializer::DeserializeUncompressedAst(
     base::Optional<uint32_t> start_offset, base::Optional<uint32_t> length, uint8_t* uncompressed_ast, size_t uncompressed_size) {
   int offset = 0;
   bool is_toplevel = true;
+  serialized_ast_ = uncompressed_ast;
```

Comment:
I feel like this is something that should probably be const and go in the constructor, but this is fine for now.